package-lock.json

```diff
@@ -388,7 +388,6 @@
             "requires": {
```

Comment:
Why did `fsevents` and `nan` get removed? We're not using them for anything anymore?

Reply:
I think we haven't been using them for some time and an npm install just updated it